docs/docsite/rst/scenario_guides/guide_aws.rst

```diff
@@ -169,32 +169,32 @@ Host Inventory
 ``````````````
 
 Once your nodes are spun up, you'll probably want to talk to them again.  With a cloud setup, it's best to not maintain a static list of cloud hostnames
-in text files.  Rather, the best way to handle this is to use the ec2 dynamic inventory script. See :ref:`dynamic_inventory`.
+in text files.  Rather, the best way to handle this is to use the aws_ec2 inventory plugin. See :ref:`dynamic_inventory`.
 
-This will also dynamically select nodes that were even created outside of Ansible, and allow Ansible to manage them.
-
-See :ref:`dynamic_inventory` for how to use this, then return to this chapter.
+Depending on how it is configured the plugin can also return instances that were created outside of Ansible and allow Ansible to manage them.
```

Comment:
I'd like to remove this ambiguity (Depending on how it is configured).  I can't think of a way to restrict inventory to just Ansible-created instances other than "be sure to tag your instances so you can filter them properly", so I think instead we can say something like: 
```suggestion
This will also select nodes that were created outside of Ansible, and allow Ansible to manage them. To limit what nodes are returned, you can use [filters](https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instances.html#options).
```